src/pages/catalog-page/product-card/types.ts

```diff
@@ -0,0 +1,7 @@
+export type ProductType = {
```

Comment:
```suggestion
export type ProductCard = {
```
или типа того. а то тайп ПродактТайп. если кто с первого раза не понял что это тайп)